### DISCUSSION TITLE
Which Grafana versions does grafana-pdf-exporter support?

### DISCUSSION THREAD

**Discussion by BinYaoDB** [artifacthub/hub#2411] [Q&A] [answered]
Hi,

Which Grafana versions does grafana-pdf-exporter support? 

thanks

**Comment by tegioz** [comment, marked as answer]
Hi @BinYaoDB 👋

For issues with the content listed on artifacthub.io it's best to contact the particular project directly. In the links section of the right panel in the package view you may find some links that can point you to the project's repository 🙂

**Reply by BinYaoDB** [reply]
OK, thanks
